dojo/tools/kiuwan_sca/parser.py

```diff
@@ -0,0 +1,89 @@
+import hashlib
+import json
+
+from dojo.models import Finding
+
+__author__ = "mwager"
+
+
+class KiuwanSCAParser:
+    SEVERITY = {
+        "-": "Low",
+        "LOW": "Low",
+        "MEDIUM": "Medium",
+        "HIGH": "High",
+        "CRITICAL": "Critical",
+        "Low": "Low",
+        "Medium": "Medium",
+        "High": "High",
+        "Critical": "Critical",
+    }
+
+    def get_scan_types(self):
+        return ["Kiuwan SCA Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Kiuwan Insights Scan in JSON format. Export as JSON using Kiuwan REST API."
+
+    def get_findings(self, filename, test):
+        data = json.load(filename)
+        dupes = {}
+
+        for row in data:
+            # if a finding was "muted" in the Kiuwan UI, we ignore it (e.g. marked as false positive)
+            if row["muted"] is True:
+                continue
+
+            finding = Finding(test=test)
+            finding.unique_id_from_tool = row["id"]
+            finding.cve = row["cve"]
+            finding.description = row["description"]
+            finding.severity = self.SEVERITY[row["securityRisk"]]
+
+            if "components" in row and len(row["components"]) > 0:
+                finding.component_name = row["components"][0]["artifact"]
+                finding.component_version = row["components"][0]["version"]
+                finding.title = finding.component_name + " v" + str(finding.component_version)
+
+            if not finding.title:
+                finding.title = row["cve"]
+
+            if "cwe" in row:
+                try:
+                    finding.cwe = int(row["cwe"].replace("CWE-", ""))
+                except Exception:
```

Comment:
Can you please cover only the exceptions that are expected? 
Probably only `KeyError` and `ValueError`

And if only actions is `pass` it might be better idea to use 
```python
import contextlib
with contextlib.suppress(...):
```

Reply:
It still might fail on `row['cwe'] = 'CVE-X'` :) But looks better.

Reply:
@kiblik updated :)